### PR TITLE
Add Meer5 to tech-docs

### PR DESCRIPTION
**Pull request by leviport**

### CODE REVIEW

File: src/models/meer5/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Meerkat (meer5)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Short Meerkat](./img/meer5-short.png)
+
+The System76 Thelio Massive is a desktop with the following specifications:
```
Comment (ahoneybun) [inline src/models/meer5/README.md:9]: Thelio Massive? Are these specs correct?

**Review by ahoneybun — CHANGES_REQUESTED** [review]

File: src/models/meer5/README.md (hunk 1 above)
Comment (leviport) [reply src/models/meer5/README.md]: Hey, meerkats can dream, can't they? 

Thanks for catching that. I was copying chunks of Massive's docs to use as a template and forgot to update that. The specs are correct.

**Comment by ahoneybun** [comment]
When I run it live using `mdbook serve` I don't see the meer5. I can see the files being loaded but the .html files are not created for that model. Do you see that @leviport ?

**Comment by leviport** [comment]
> When I run it live using mdbook serve I don't see the meer5. I can see the files being loaded but the .html files are not created for that model. Do you see that @leviport ?

Whoops, I think I know what the issue is. Just a moment.

**Review by ahoneybun — APPROVED** [review]
LGTM!

**Pull request merged**